tests/data/MerchFactory.ts

```diff
@@ -15,7 +16,21 @@ export class MerchFactory {
   }
 
   public static createOptions(n: number): MerchandiseItemOptionModel[] {
-    return FactoryUtils.create(n, MerchFactory.fakeOption);
+    const options = FactoryUtils.create(n, MerchFactory.fakeOption);
+    if (n === 1) return options;
+
+    const type = faker.datatype.hexaDecimal(10);
+    return options.map((o) => {
+      [o.metadata] = MerchFactory.optionMetadataWith({ type });
```

Comment:
What're the brackets for?

Reply:
`optionMetadataWith` returns an array (e.g. `T[]`), the brackets make it so that `o.metadata` is of type `T`.